internal/backend/runtime/omni/controllers/omni/internal/configpatch/configpatch.go

```diff
@@ -54,8 +54,11 @@ func (h *Helper) Get(machine *omni.ClusterMachine, machineSet *omni.MachineSet)
 	for patch := range clusterPatchList.All() {
 		machineSetName, machineSetOk := patch.Metadata().Labels().Get(omni.LabelMachineSet)
 		clusterMachineName, clusterMachineOk := patch.Metadata().Labels().Get(omni.LabelClusterMachine)
+		_, machineOk := patch.Metadata().Labels().Get(omni.LabelMachine)
 
 		switch {
+		// skip it, it's a machine patch (and it also has cluster label)
```

Comment:
this is a breaking (?) change potentially, as it skips invalid patches which have both Machine & Cluster labels